Conventional annotation of transcription start sites (TSSs) rely on low-throughput technologies such as EST/cDNA alignment and 5 rapid amplification of cDNA ends (RACE). In recent years, cap-trapping protocols such as CAGE \cite{Takahashi2012} aim to isolate pol-\Rnnum{2} mRNA transcript and sequence the capped ends (TSS tags) with nucleotide resolution. 
By mapping TSSs back to the genome, we can define promoter regions for all genes expressed under the experimental conditions. We then performed \textit{de novo} motif discovery based on this set of mapped TSSs, obtaining overrepresented sequence of canonical core promoter elements.
